src/components/topNav/TopNav.module.css

```diff
@@ -1,22 +1,52 @@
-.ul {
-  display: flex;
+.primary,
+.secondary {
+  width: fit-content;
 }
 
-.ul > * {
-  flex: 1;
+.ul {
+  margin: unset;
+  display: flex;
+  padding-inline-start: unset;
+  align-items: center;
 }
 
 .li {
   list-style-type: none;
-  padding-block-start: var(--conduction-navbar-padding);
-  padding-block-end: var(--conduction-navbar-padding);
 }
 
-.li > * {
+.li > a {
```

Comment:
It's best to never target a element directly (in this case you're targeting `a`). Best to add a className to the anchor element and thereafter target the className.